package.json

```diff
@@ -70,9 +55,9 @@
     "patternfly-react": "^0.8.1",
     "prop-types": "^15.6.0",
     "raf": "^3.4.0",
-    "react": "^16.0.0",
+    "react": "^16.1.1",
```

Comment:
probably should be done in a different pr.

Reply:
:+1:

Reply:
After rethinking about it, we introducing yarn so it will get saved to my `yarn.lock` as `16.1.1` anyway.
Would you like to force it to be `16.0.0` on the `yarn.lock` file?